webpack.config.js

```diff
@@ -0,0 +1,179 @@
+const webpack = require("webpack");
+const { CleanWebpackPlugin } = require("clean-webpack-plugin");
+const FileManagerPlugin = require("filemanager-webpack-plugin");
+const XMLPlugin = require("xml-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const UglifyJsPlugin = require("uglifyjs-webpack-plugin");
+
+process.traceDeprecation = true;
+process.noDeprecation = true;
+
+const MODES = {
+    DEV: "development",
+    PROD: "production"
+};
+
+const isDev = process.env.MODE === MODES.DEV;
+
+const widgetDir = `BootstrapTooltip/widget`;
+const widgetUIDir = `${widgetDir}/ui`;
+
+const widgetXMLFiles = [
+    {
+        template: "./src/package.xml",
+        filename: "package.xml",
+        data: {
+            VERSION: process.env.npm_package_version
+        }
+    },
+    {
+        template: "./src/BootstrapTooltip.xml",
+        filename: "BootstrapTooltip/BootstrapTooltip.xml"
+    },
+    {
+        template: "./src/BootstrapTooltipContext.xml",
+        filename: "BootstrapTooltip/BootstrapTooltipContext.xml"
+    }
+];
+
+module.exports = {
+    mode: isDev ? MODES.DEV : MODES.PROD,
+    target: "web",
+    devtool: isDev ? "eval-source-map" : false,
+    entry: {
+        BootstrapTooltipContext: "./src/BootstrapTooltipContext.js",
+        BootstrapTooltip: "./src/BootstrapTooltip.js"
+    },
+    watch: isDev,
+    watchOptions: {
+        ignored: /node_modules/
+    },
+    output: {
+        path: `${__dirname}/dist/BootstrapTooltip`,
+        filename: `${widgetDir}/[name].js`,
+        libraryTarget: "amd"
+    },
+    optimization: {
+        minimizer: [

```

Comment:
Discuss for mono repo, if dojo widgets should be minified as source maps are not supported for bundled widgets.js